x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_dashboard.tsx

```diff
@@ -41,6 +42,10 @@ import { BenchmarksSection } from './dashboard_sections/benchmarks_section';
 import { CSPM_POLICY_TEMPLATE, KSPM_POLICY_TEMPLATE } from '../../../common/constants';
 import { cspIntegrationDocsNavigation } from '../../common/navigation/constants';
 
+const REFETCH_INTERVAL_MS = 20000;
```

Comment:
any particular reason to go with 20s? my suggestion would be to go with 10 or even 5 seconds.
this refetch only happens on the initial state so it won't be spamming calls for too long, we can also use set reactQuery to only make the calls when the window is on focus.

Reply:
I went with the 20s to be consistent with `NoFindingsStates` component https://github.com/elastic/kibana/blob/main/x-pack/plugins/cloud_security_posture/public/components/no_findings_states.tsx#L35 but your question also shows that then I need to make it explicit to move it to a shared constant. 
Regarding the particular 20s value, I think we can down to 10s, don't see an issue with that. The plugin installation and agent deployment take some time, so it feels like 5s would be a bit too low, but this feeling is not backed by any evidence or number

Reply:
updated to 10s and moved to a shared constant